File: app/src/main/java/dev/ragnarok/fenrir/api/model/VkApiStickerSetsData.java
package dev.ragnarok.fenrir.api.model;

import androidx.annotation.Nullable;

import com.google.gson.annotations.SerializedName;


public class VKApiStickerSetsData {

    @Nullable
    @SerializedName("recent")
    public Items<VKApiSticker> recent;

    @Nullable
    @SerializedName("sticker_pack")
    public Items<VKApiStickerSet.Product> sticker_pack;

}

File: app/src/main/kotlin/dev/ragnarok/fenrir/api/model/VKApiStickerSetsData.kt
package dev.ragnarok.fenrir.api.model

import com.google.gson.annotations.SerializedName

class VKApiStickerSetsData {
    @SerializedName("recent")
    var recent: Items<VKApiSticker>? = null

    @SerializedName("sticker_pack")
    var sticker_pack: Items<VKApiStickerSet.Product>? = null
}